libraries/scrcpy/src/codec/h264.ts

```diff
@@ -61,7 +97,7 @@ class BitReader {
  * This methods returns a generator, so it can be stopped immediately
  * after the interested NAL unit is found.
  */
-export function* splitH264Stream(buffer: Uint8Array): Generator<Uint8Array> {
+export function* h264SplitNalu(buffer: Uint8Array): Generator<Uint8Array> {
```

Comment:
Find out does H.265 use the same NALU structure. Rename it to `iterateNalu` (remove `h264` part)